testing/features/contact_details.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Contact Details component
+
+  The Contact Details component provides users with full contact information
+  so that they can contact the relevant authority via different methods
+
+  Background:
+    Given a Contact Details component is on the page
+
+  Scenario: Validate initial state
+    Then the header says "Contact details"
```

Comment:
The header will be the contact name in bold, eg "Bail for Immigration Detainees".

Reply:
This is the page title header (h1). And on this page it says "Contact details" - Should it say something different?

Reply:
That is the styleguide header, the component itself is within the red box. Just like all the other ones.

![contact details](https://user-images.githubusercontent.com/12530094/92104029-aed2f080-edd8-11ea-9e10-b189e873c60a.jpg)

Reply:
So the first paragraph which is bold, should always be a title for where to contact the people?

Reply:
It really is up to what the content editor has put in. If they make the first line bold then fine, if not then it is what it is. This will be down to the guidelines that content editors get. The design system itself has no code to enforce styling.

Reply:
Hi Panos, just realised this is still requiring changes. What needs editing? This line you've highlighted is the assertion at the top of the page. Not the one you've highlighted? Do I need to change the assertion/s below (1+ paragraph and bold), this was on the ticket in JIRA?

Reply:
The check that the title is "Contact details" is irrelevant. The component is the one I highlighted. You're effectively checking part of the styleguide UI, not the component.